test/Base_Tests/src/Semantic/Conversion_Spec.enso

```diff
@@ -617,6 +617,22 @@ add_specs suite_builder =
             r = [r1, r2]
             r.to_text . should_contain "Foo.Value 4"
 
+        group_builder.specify "Convert and catch" <|
+            fn x = x+2
```

Comment:
What is the purpose of this declaration?

Reply:
I need a function to pass as a value to the `convert_and_catch` function. It mimics [the example](https://github.com/enso-org/enso/issues/10236#issuecomment-2162746500) from the bug report.